html-compile.js

```diff
@@ -1,20 +1,20 @@
 const pug = require('pug');
 const fs = require('fs');
 const variables = require('./index.json');
-const package = require('./package.json');
+const pkg = require('./package.json');
```

Comment:
```suggestion
const package = require('./package.json');
```

Reply:
The linter fails on "package" because it's a reserved word. Hence writing "pkg"